Console/Command/RegenerateProductUrlCommand.php

```diff
@@ -113,111 +136,107 @@ protected function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-
-        try {
-            $this->state->setAreaCode('adminhtml');
-        } catch (\Magento\Framework\Exception\LocalizedException $e) {
-            // intentionally left empty
-        }
-
         $this->output = $output;
+        $this->state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
```

Comment:
Try catch mist en is verplicht om dit toe te voegen om problemen te voor komen, is toegevoegd om de error areacode already set te voorkomen